scripts/elf_helper.py

```diff
@@ -25,8 +25,13 @@ def subsystem_to_enum(subsys):
     return "K_OBJ_DRIVER_" + subsys[:-11].upper()
 
 
-def kobject_to_enum(ko):
-    return "K_OBJ_" + ko[2:].upper()
+def kobject_to_enum(kobj):
+    if kobj.startswith("k_") or kobj.startswith("_k_"):
```

Comment:
`startswith()` supports testing more than one string with a tuple btw: `if kobj.startswith(("k_", "_k_")): ...`

`_k_foo` will turn into `K_OBJ__FOO`, but that might be intended.

Reply:
It currently is, yes.